rules/opentitan_test.bzl

```diff
@@ -392,5 +392,4 @@ def opentitan_functest(
     native.test_suite(
         name = name,
         tests = all_tests,
-        tags = ["test_suite"],
```

Comment:
Yeah, I screwed up here and assumed the tags arg for test suites worked like tests, but they don't.